### PR TITLE
[SIG-3392] Hides the  'delen' button when the the split status is s

### PR DESCRIPTION
**Pull request by aditudorache**
This PR hides the delen button also when the status is s

### CODE REVIEW

File: src/signals/incident-management/containers/IncidentDetail/components/DetailHeader/index.js [hunk 1]
```diff
@@ -90,7 +90,7 @@ const DetailHeader = () => {
   const location = useLocation();
 
   const showSplitButton = useMemo(() => {
-    if (incident.status.state === 'o' || incident.status.state === 'a') return false;
+    if (['o', 'a', 's'].some(value => value === incident.status.state)) return false;
```
Comment (jpoppe) [inline src/signals/incident-management/containers/IncidentDetail/components/DetailHeader/index.js:93]: I would use `includes`  instead of `some` here.

**Review by jpoppe — COMMENTED** [review]
I left a note, looks good!

File: src/signals/incident-management/containers/IncidentDetail/components/DetailHeader/index.js (hunk 1 above)
Comment (aditudorache) [reply src/signals/incident-management/containers/IncidentDetail/components/DetailHeader/index.js:93]: Thanks, I did that first, but further in this file is this construction with `some` used so I prefer to keep it consistent with the rest of the code. Performance wise makes no difference.

Comment (MauritsR) [reply src/signals/incident-management/containers/IncidentDetail/components/DetailHeader/index.js:93]: I'd use `includes` as well - if that's the initial thought for all 3 of us, it makes sense to use `includes` here and change the other occurance of `some` to `includes` as well

**Review by MauritsR — APPROVED** [review]

**Pull request merged**